consensus/e2e_tests/utils_test.go

```diff
@@ -589,6 +591,24 @@ func baseRpcMock(t *testing.T, _ modules.EventsChannel) *mockModules.MockRPCModu
 	return rpcMock
 }
 
+func baseIbcMock(t *testing.T, _ modules.EventsChannel) *mockModules.MockIBCModule {
```

Comment:
Would you be up for moving this to an `/internal/testutil/ibc` package? 

While both are still unpolished/incomplete, I think there should be enough context in [Testutils > Mocks](https://pocketnetwork.notion.site/Testutils-9cba9010e18447248e9daa8a3b87e3f2?pvs=25) and examples in #797 to land it smoothly. Wdyt?

Reply:
@bryanchriswhite will look into this now 👍🏼